sqlmesh/core/console.py

```diff
@@ -1904,10 +1905,37 @@ def show_row_diff(
                     # Create a table with the joined keys and comparison columns
                     column_table = row_diff.joined_sample[keys + [source_column, target_column]]
 
-                    # Filter out identical-valued rows
+                    def compare_cells(x: t.Any, y: t.Any) -> bool:
```

Comment:
This is too many level of nestedness. Let's make this top level with `_`

Reply:
@themisvaltinos I'd double-check if we already have similar logic in `tests/definition.py`. My expectation is we're probably normalizing there as well because we deal with dataframes, so perhaps there's room for refactoring here?